pod/perl5380delta.pod

```diff
@@ -2175,6 +2175,30 @@ space is actually allocated. C<SSCHECK()> is now an alias for C<SSGROW()>.
 
 =back
 
+=head1 Known Problems
```

Comment:
I don't know why but "Known Issues" sounds more familiar 🙂

Reply:
In general, I’d tend to agree, but in perldelta, “known problems” is the usual terminology:

https://github.com/Perl/perl5/blob/blead/Porting/perldelta_template.pod#known-problems

Reply:
`=head1 KNOWN BUGS` would be the traditional heading. level-1 headings are all-caps.

Reply:
Actually I think `=heads1 BUGS` is the traditional heading in Perl reference documents other than perldelta, see e. g. [perlre](https://perldoc.perl.org/5.38.2/perlre#BUGS) or [perlebcdic](https://perldoc.perl.org/5.38.2/perlebcdic#BUGS). But I’m happy with “known bugs” if you prefer it.